### ISSUE TITLE
Inputs to the layers that project to logits

### ISSUE THREAD

**Issue by fkarasek** [google-research/fixmatch#29]
Hello again, 

You have provided some very good information in my previous post. But I am still eager to learn. I hope it is okay to bother you once more. I intend on accessing the inputs to the layers that project to logits. I want to save these tensors in a file. I was wondering if you can point me in the right direction in how to do this. So far I have tried "f.write(tf.strings.as_string(logits_x)"
but Im sure there must be a better way to do this. Thanks again

**Comment by fkarasek** [comment]
Okay I think I got it. It took me some time to understand tensorflow. I basically just loaded the model checkpoint and meta files, then listed the tensors and selected the dense/kernel tensor - saved it into a .npy file
